tests/test_nyt.py

```diff
@@ -227,11 +227,13 @@ def test_to_customlist(self):
 
         jan_17 = datetime.datetime(2015, 1, 17).replace(tzinfo=self.api.TIME_ZONE)
         assert (True ==
-            all([x.first_appearance == jan_17 for x in custom.entries]))
+            all([x.first_appearance.replace(tzinfo=self.api.TIME_ZONE) ==
```

Comment:
`x.first_appearance` gets its timezone information from the database since the list entries are queried from the database. It'll appear as `datetime.datetime(2015, 1, 17, 0, 0, tzinfo=psycopg2.tz.FixedOffsetTimezone(offset=0, name=None))` so that's why this test fails. But not sure if this is the _best_ way to check this.

Reply:
Yeah, I don't get it -- first_appearance should be set at a date that represents midnight Eastern time, not midnight UTC.

Reply:
I updated `jan_17` to `datetime.datetime(2015, 1, 17, tzinfo=dateutil.tz.tzoffset("", 0))` which gives a better representation of what's coming from the database.

Reply:
See my later comment -- first_appearance was set to midnight UTC because the timezone info got wiped when we converted datetime to date.